src/schema/asset_uploads/create_asset_request_mutation.ts

```diff
@@ -95,6 +95,7 @@ export default mutationWithClientMutationId({
     { rootValue: { createNewGeminiAssetLoader } }
   ) => {
     if (!createNewGeminiAssetLoader) return null
-    return createNewGeminiAssetLoader({ name, acl })
+    console.log(name, acl)
```

Comment:
we should remove this 🙌

Reply:
Should we really remove it? I like my console logs!

:trollface: :trollface: :trollface: